src/hooks/useNotificationModal.ts

```diff
@@ -161,6 +164,9 @@ export const useNotificationModal = ({
           () => ""
         );
         // validator returns an error message or an empty string
+        if (validator == null) {
+          return "";
+        }
```

Comment:
I'm guessing that you may be able to remove this addition as well. I assume it's another problem that you initially ran into but it went away? I don't see any errors when I remove it 🤔